app/uk/gov/hmrc/apigatekeeperapisfrontend/views/ApiListPage.scala.html

```diff
@@ -15,13 +15,30 @@
  *@
 
 @import uk.gov.hmrc.apiplatform.modules.gkauth.domain.models.LoggedInRequest
+@import uk.gov.hmrc.apiplatform.modules.apis.domain.models._
 
 @this(layout: Layout)
 
-@()(implicit request: LoggedInRequest[_])
+@(apis: List[ApiData])(implicit request: LoggedInRequest[_])
 
     @layout(pageTitle = Some("APIs")){
       <h1 class="govuk-heading-l">
           APIs
       </h1>
-    }
+    <table class="govuk-table responsive-table" data-module="moj-sortable-table">
```

Comment:
Did you consider using the Twirl component for the table? See [example](https://github.com/hmrc/play-frontend-govuk-examples/blob/main/src/test/play-2/twirl/uk/gov/hmrc/govukfrontend/views/examples/table/columnWidths.scala.html).